src/routers/moviesRouter.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const router = express.Router()
+
+// Controllers
+const {
+  getAllMovies,
+  createMovie,
+  getMovieById,
+  updateMovieById
+} = require('../controllers/movies')
+
+// Retrieve a list of movies
+router.get('/', getAllMovies)
```

Comment:
The comments for these routers aren't adding much imo. I can already tell what `getAllMovies` is likely to do just from its name.